apps/nextjs-website/src/lib/strapi/__tests__/webinars.test.ts

```diff
@@ -0,0 +1,189 @@
+      attributes: {
+        title: 'title',
+        description: 'description',
+        slug: 'slug-1',
+        textContent: [
+          {
+            type: 'paragraph',
+            children: [
+              {
+                type: 'text',
+                text: 'textContent',
+              },
+            ],
+          },
+        ],
+        playerSrc: 'https://playerSrc/1',
+        startDatetime: '2024-03-25T23:00:00.000Z',
+        endDatetime: '2024-03-27T23:00:00.000Z',
+        isVisibleInList: true,
+        subscribeParagraphLabel: 'subscribeParagraphLabel',
+        createdAt: '2024-03-26T17:12:29.910Z',
+        updatedAt: '2024-03-26T17:12:31.925Z',
+        publishedAt: '2024-03-26T17:12:31.921Z',
+        locale: 'it',
+        coverImage: {
+          data: {
+            id: 1,
+            attributes: {
+              name: 'a-image.png',
+              alternativeText: null,
+              caption: null,
+              width: 1440,
+              height: 495,
+              hash: 'image_52d70c707f',
+              ext: '.png',
+              mime: 'image/png',
+              url: '/uploads/image_52d70c707f.png',
+            },
+          },
+        },
+        webinarSpeakers: {
+          data: [
+            {
+              id: 1,
+              attributes: {
+                name: 'name',
+                jobTitle: 'jobTitle',
+                description: [
+                  {
+                    type: 'paragraph',
+                    children: [
+                      {
+                        type: 'text',
+                        text: 'desc',
+                      },
+                    ],
+                  },
+                ],
+                createdAt: '2024-03-26T17:10:49.863Z',
+                updatedAt: '2024-03-26T17:12:37.276Z',
+                publishedAt: '2024-03-26T17:12:37.274Z',
+                locale: 'it',
+                avatar: {
+                  data: {
+                    id: 1,
+                    attributes: {
+                      name: 'a-image.png',
+                      alternativeText: null,
+                      caption: null,
+                      width: 1440,
+                      height: 495,
+                      hash: 'image_52d70c707f',
+                      ext: '.png',
+                      mime: 'image/png',
+                      url: '/uploads/image_52d70c707f.png',
+                    },
+                  },
+                },
+              },
+            },
+          ],
+        },
+        relatedLinks: {
+          id: 2,
+          title: 'title',
+          links: [
+            {
+              id: 13,
+              text: 'text',
+              href: 'href',
+              target: null,
+            },
+          ],
+        },
+      },
+    },
+  ],
+  meta: {
+    pagination: {
+      page: 1,
+      pageSize: 25,
+      pageCount: 1,
+      total: 1,
+    },
+  },
+});
+
+const makeStrapiResponseWithNullsJson = () => ({
```

Comment:
Is this scenario useful? It is going to improve the code coverage?

Reply:
I don't think that this test improve test coverage but i added this test following the example in `apps/nextjs-website/src/lib/strapi/__tests__/homepage.test.ts`. Maybe they both here and in homepage can be removed :thinking: